x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/utils.ts

```diff
@@ -67,10 +76,13 @@ export const transformTags = (tags: string[]): string[] => {
 
 // Transforms the data but will remove any null or undefined it encounters and not include
 // those on the export
-export const transformAlertToRule = (alert: RuleAlertType): Partial<OutputRuleAlertRest> => {
+export const transformAlertToRule = (
+  alert: RuleAlertType,
+  ruleStatus?: SavedObject<IRuleSavedAttributesSavedObjectAttributes>
+): Partial<OutputRuleAlertRest> => {
   return pickBy<OutputRuleAlertRest>((value: unknown) => value != null, {
     created_at: alert.params.createdAt,
-    updated_at: alert.params.updatedAt,
+    updated_at: ruleStatus?.attributes.statusDate ?? alert.params.updatedAt,
```

Comment:
Mistake maybe here? Just because a rule has a new status, I don't think that means the rule its self is updated? Otherwise won't we continuously show the rule being updated when its not? Every time someone goes to edit a rule or change it while its running it would show that the rule is always updated?

I would for now keep this `updated_at` separate from the `status_date` `status_date`